.github/workflows/ci.yml

```
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+    pull_request: ~
+    push:
+        branches:
+            - main
+        tags:
+            - '*'
+    schedule:
+        - cron: 0 13 * * MON,THU
+
```

```suggestion
on:
    pull_request: ~
    push: ~
```

`schedule` is very specific IMO, triggering on every push and on every pull request seems like a good default to me.

---

See [51bc1b9](https://github.com/contao/skeleton-bundle/pull/11/commits/51bc1b9f6fc4f13ca7a865caec4ac807716668bd)